vision/use-cases/Advanced_prompting_for_image_generation_on_Imagen2.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>Does this need to be in JSON format since it's just being provided by the user directly?</p><p> </p><p>Could these be made into constants/parameters?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
These are deliberately kept as JSON, so that if a user connects this with a UI, JSON is the best way to communicate between the 2